cfpb/cfgov-refresh
# Paying for College: Update analytics script jquery version

Supersedes https://github.com/cfpb/cfgov-refresh/pull/5656/

## Changes

- Update `jquery` for college analytics script

## Testing

1. Pull branch and run `./frontend.sh`
2. Visit http://localhost:8000/paying-for-college/compare-financial-aid-and-college-cost/?debug-gtm=true
3. Click buttons in the tool and see that analytics are logged to the console.
